src/main/java/pages/UtilitiesAka.java

```diff
@@ -0,0 +1,38 @@
+
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import javax.xml.xpath.XPath;
+
+
+public final class UtilitiesAka {
+
+    private static WebDriver driver;
+    private static int timeOut = 10;
+
+    public UtilitiesAka(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+
+
+    public String ElementAssertAndFill(String text, String invullen){
```

Comment:
Zorg ervoor dat je parameter namen zoals nu text en invullen een logisch naam krijgen. Vanuit je IDE kun je dan in de methode waarin je deze helper aanroept dan makkelijker zien wat je aan deze methode moet meegeven. Het is bijv lastig om te zien dat "text" in dit geval eigenlijk om een xpath gaat.